parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -445,25 +462,27 @@ private void enqueueEvent(Map<String, Object> event) {
         }
     }
 
-    /*!  \brief Flush events to Parsely.
-     *
-     *  Empties the event queue and sends the appropriate requests to Parsely.
-     *  Called automatically after a number of seconds determined by `flushInterval`.
-     *
-     *  To make sure all of the queued events are flushed to Parse.ly's servers,
-     * include a call to `flushEventQueue` in your main activity's `onDestroy()` method.
+    /**
+     * Flush events to Parsely.
+     * <p>
+     * Empties the event queue and sends the appropriate requests to Parsely.
+     * Called automatically after a number of seconds determined by {@link #getFlushInterval()}.
+     * <p>
+     * To make sure all of the queued events are flushed to Parse.ly's servers,
+     * call this method in your main activity's `onDestroy()` method.
```

Comment:
This is the only place I've modified the content slightly. Previous wording was confusing to me and was referencing the method it's commenting on, which, it seems, is not really correct in Javadoc convention:

<img width="521" alt="image" src="https://github.com/Parsely/parsely-android/assets/5845095/6ef7898c-8049-4f82-9d68-f9d7e4f7023c">